bluetooth/src/androidLibMain/kotlin/scanner/Scanner.kt

```diff
@@ -50,9 +56,9 @@ actual class Scanner internal constructor(
     permissions: Permissions,
     connectionSettings: ConnectionSettings,
     autoRequestPermission: Boolean,
-    autoEnableBluetooth: Boolean,
```

Comment:
Don't want to block this PR out of this but, since this is the android specific version of the scanner, we can also `autoEnableSensors` split into 2 booleans: `autoEnableBluetooth` (passing it down) and `autoEnableLocation` (used only in this class)
(Although, on Android 12 this flag would not make sense once again)

The added value would be to have more control over what happens when gps toggles.. some projects that uses location for more than just BT purposes might already have some custom code handling this scenario and this could be a source of interference.

Furthermore, imho they should default to `false`